### ISSUE TITLE
11/06/24_<NYP><Marker><Android><Master> For blind user not getting ride ended notification sound (Parent id_1980)

### ISSUE THREAD

**Issue by Vinaykm999** [nammayatri/nammayatri#7240]
#1980 

[For blind user not getting ride ended notification sound ](https://drive.google.com/file/d/19cQyppAJMMvB2rPAPmx3Z_AvnPe8wis4/view?usp=sharing)

1. Make user as blind 
2. Start ride
3. End ride
4. Verify the notification sound 

Expected: Ride end notification should come

**Comment by PraveenGongada** [comment]
working fine in `main` branch, please check in next apk

**Comment by Vinaykm999** [comment]
No. It is not device specific, for rest sounds like ride assigned and started i'm getting.

**Comment by Vinaykm999** [comment]
This issue is still exist, Hence reopening - [PFA](https://drive.google.com/file/d/1FfKbiNzJaGmxP3zF6VhkZkFbAKWEY9qc/view?usp=sharing)
